tensorflow/core/common_runtime/BUILD

```diff
@@ -1040,9 +1045,13 @@ cc_library(
 cc_library(
     name = "mkl_tfconversion_pass",
     srcs = ["mkl_tfconversion_pass.cc"],
-    hdrs = ["mkl_tfconversion_pass.h"],
+    hdrs = [
+        "mkl_tfconversion_pass.h",
+        "//tensorflow/core/graph:mkl_graph_util_header",
+    ],
     copts = tf_copts(),
     deps = [
+        ":function",
```

Comment:
This seems to be causing a circular dependency. Try building `//tensorflow/c:c_api_test` and you should see something like:

```
.-> //tensorflow/core/common_runtime:core_cpu
|   //tensorflow/core/common_runtime:core_cpu_internal
|   //tensorflow/core/common_runtime:core_cpu_impl
|   //tensorflow/core/common_runtime:mkl_tfconversion_pass
`-- //tensorflow/core/common_runtime:core_cpu
```

Reply:
@jaingaurav Thanks. I am not seeing this when testing CPU (with and without MKL) on Linux. Can you provide more info on what platform the test is failing.  One of my previous commits in this PR had the error on Windows builds only, but the newer commit should have fixed it.